docs/learn/smart-contract-developers/create-nft-collection.md

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_label: '🗃 Create an NFT Collection'
+sidebar_position: 4
+---
+
+# Create an NFT Collection Using LSP8
+
+This tutorial goes over creating a collection of collectibles, where each item is unique and has its own design.
+
+:::info
+⌨️ The full code of this example can be found in 👾 [here](https://github.com/CJ42/LUKSO-Hardhat-template).
+:::
+
+## Setup
+
+Make sure you have the following dependencies installed:
+
+- [`ethers.js`](https://github.com/ethers-io/ethers.js/) (alternatively you can use [`web3.js`](https://github.com/web3/web3.js))
+- [`@lukso/lsp-smart-contracts`](https://github.com/lukso-network/lsp-smart-contracts/)
+
+## Create the Smart Contracts
+
+When creating digital asset smart contracts on LUKSO, you will need to specify the token type and the following data keys in the ERC725Y storage. There are three types that you can define:
+
+- 0 = Token
+- 1 = NFT
+- 2 = Collection
```

Comment:
Does the number have a meaning? If so, I would suggest putting it in code style.

```suggestion
- `0` = Token
- `1` = NFT
- `2` = Collection
```